FontAwesome5Pro.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * FontAwesome5Pro icon set component.
+ * Usage: <FontAwesome5Pro name="icon-name" size={20} color="#4F8EF7" />
+ */
+
+import {FA5iconSet} from './FontAwesome5';
```

Comment:
Not sure if the linter caught it but, should be spaces around the curly braces like this: `{ FA5iconSet }

Reply:
No, I didn't get a warning but I'll fix it 👍🏻